common/app/templates/inlineJS/blocking/analytics.scala.js

```diff
@@ -130,6 +130,43 @@ try {
         s.prop20    = tpA[2] + ':' + tpA[1];
         s.eVar20    = 'D=c20';
 
+        var participationsKey = 'gu.ab.participations';
+
+        try {
+            var participations = window.localStorage.getItem(participationsKey);
+
+            var abTestsParticipations = makeOmnitureABTag(participations);
+
+            s.eVar51    = abTestsParticipations;
+            s.list1     = abTestsParticipations;
+
+            // This is set globally so we can check if the use ab test participations change once the ab test runs.
+            // If it does, we fire a second tracking call in modules/analytics/omniture.js
+            guardian.config.abTestsParticipations = abTestsParticipations;
+        } catch (e) { }
+
+        function makeOmnitureABTag(currentParticipations) {
+            var participations = JSON.parse(currentParticipations);
+            var tag = [];
+
+            for (var key in participations.value) {
+                tag.push(['AB', key, participations.value[key].variant].join(' | '));
```

Comment:
the result of json.parse isn't part of any prototype chain so we don't need any `hasOwnProperty` check.
![screen shot 2016-02-04 at 10 59 11](https://cloud.githubusercontent.com/assets/1064734/12813197/6618c3b4-cb2e-11e5-97ce-90669dd044a2.png)